infrastructure/k8s-config/clusters/kit-infrastructure/perfdash/perfdash.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: perfdash
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: perfdash-config
+  namespace: perfdash
+  labels:
+    app.kubernetes.io/name: perfdash
+    app.kubernetes.io/component: config
+    app.kubernetes.io/part-of: perfdash
+data:
+  jobs.yaml: |
+    periodics:
+      - name: "kit-eks-1k"
+        tags:
+          - "perfDashPrefix: 1K test"
+          - "perfDashJobType: performance"
+      - name: "kit-eks-2k"
+        tags:
+        - "perfDashPrefix: 2k test"
+        - "perfDashJobType: performance"
+      - name: "kit-eks-3k"
+        tags:
+          - "perfDashPrefix: 3K test"
+          - "perfDashJobType: performance"
+      - name: "kit-eks-36k"
+        tags:
+        - "perfDashPrefix: 36k test"
+        - "perfDashJobType: performance"
+      - name: "kit-eks-4k"
+        tags:
+        - "perfDashPrefix: 4k test"
+        - "perfDashJobType: performance"
+      - name: "kit-eks-5k"
+        tags:
+          - "perfDashPrefix: 5K test"
+          - "perfDashJobType: performance"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: perfdash
+  namespace: perfdash
+  labels:
+    app.kubernetes.io/name: perfdash
+    app.kubernetes.io/version: "2.44"
+    app.kubernetes.io/component: dashboard
+    app.kubernetes.io/part-of: perfdash
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: perfdash
+      app.kubernetes.io/version: "2.44"
+      app.kubernetes.io/component: dashboard
+      app.kubernetes.io/part-of: perfdash
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: perfdash
+        app.kubernetes.io/version: "2.44"
+        app.kubernetes.io/component: dashboard
+        app.kubernetes.io/part-of: perfdash
+    spec:
+      containers:
+        - name: perfdash
+          image: gcr.io/k8s-staging-perf-tests/perfdash:2.44
+          command:
+            - /perfdash
+            - --www=true
+            - --address=0.0.0.0:8080
+            - --builds=20
+            - --force-builds
+            - --mode=s3
+            - --aws-region=us-west-2
+            - --configPath=/etc/config/jobs.yaml
+            - --logsBucket=kit-eks-scalability
+            - --logsPath=
```

Comment:
any reason for having this here with empty value ?

Reply:
It appears upstream has another layer called `logs` in the hierarchy. We don't have that and it's not necessary for our use case. So we have it as an empty value.

Reply:
cool, i see it here - https://github.com/kubernetes/perf-tests/blob/master/perfdash/perfdash.go#L52